app/experimenter/nimbus-ui/src/components/Summary/TableSummary/index.test.tsx

```diff
@@ -107,6 +108,60 @@ describe("TableSummary", () => {
     });
   });
 
+  describe("renders 'Risk mitigation question' rows as expected", () => {
+    it("when not set", () => {
+      const { experiment } = mockExperimentQuery("demo-slug", {
+        riskRevenue: null,
+        riskBrand: null,
+        riskPartnerRelated: null,
+      });
+      render(<Subject {...{ experiment }} />);
+      expect(
+        screen.getByTestId("summary-risk-mitigation-brand"),
+      ).toHaveTextContent(`${RISK_QUESTIONS.BRAND} — Not set`);
+      expect(
+        screen.getByTestId("summary-risk-mitigation-partner"),
+      ).toHaveTextContent(`${RISK_QUESTIONS.PARTNER} — Not set`);
+      expect(
+        screen.getByTestId("summary-risk-mitigation-revenue"),
+      ).toHaveTextContent(`${RISK_QUESTIONS.REVENUE} — Not set`);
+    });
+    it("when set to false", () => {
+      const { experiment } = mockExperimentQuery("demo-slug", {
+        riskRevenue: false,
+        riskBrand: false,
+        riskPartnerRelated: false,
+      });
+      render(<Subject {...{ experiment }} />);
+      expect(
+        screen.getByTestId("summary-risk-mitigation-brand"),
+      ).toHaveTextContent(`${RISK_QUESTIONS.BRAND} — No`);
+      expect(
+        screen.getByTestId("summary-risk-mitigation-partner"),
+      ).toHaveTextContent(`${RISK_QUESTIONS.PARTNER} — No`);
+      expect(
+        screen.getByTestId("summary-risk-mitigation-revenue"),
+      ).toHaveTextContent(`${RISK_QUESTIONS.REVENUE} — No`);
+    });
+    it("when set to true", () => {
+      const { experiment } = mockExperimentQuery("demo-slug", {
+        riskRevenue: true,
+        riskBrand: true,
+        riskPartnerRelated: true,
+      });
+      render(<Subject {...{ experiment }} />);
+      expect(
+        screen.getByTestId("summary-risk-mitigation-brand"),
+      ).toHaveTextContent(`${RISK_QUESTIONS.BRAND} — Yes`);
+      expect(
+        screen.getByTestId("summary-risk-mitigation-partner"),
+      ).toHaveTextContent(`${RISK_QUESTIONS.PARTNER} — Yes`);
+      expect(
+        screen.getByTestId("summary-risk-mitigation-revenue"),
+      ).toHaveTextContent(`${RISK_QUESTIONS.REVENUE} — Yes`);
```

Comment:
Suggestion: we could DRY up all three of the No/Yes/Not Set tests by doing something like:

```js
  for (const value of Object.values(RISK_QUESTIONS)) {
    expect(screen.getByText(value, { exact: false })).toHaveTextContent(
      "Yes",
    );
  }
```